### PR TITLE
BlankLineAfterNamespaceFixer - do not remove indent, handle comments

### PR DESCRIPTION
**Pull request by SpacePossum**
prove of https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4130

### CODE REVIEW

**Comment by kubawerlos** [comment]
What about:
```php
<?php
namespace Foo; /** comment 1 */              /**
 * comment 2
 */
    use Bar;
```
Should we add new line before `use` or before `comment 2` (as it has break line itself)?

**Comment by SpacePossum** [comment]
I think it would be fine to not fix in that case, sometimes people write stuff we cannot make look better :(

File: src/Fixer/PhpTag/BlankLineAfterOpeningTagFixer.php [hunk 1]
```diff
@@ -85,8 +85,12 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
             $tokens[0] = new Token([$token->getId(), rtrim($token->getContent()).$lineEnding]);
         }
 
-        if (!$tokens[1]->isWhitespace() && false === strpos($tokens[1]->getContent(), "\n")) {
-            $tokens->insertAt(1, new Token([T_WHITESPACE, $lineEnding]));
+        if (false === strpos($tokens[1]->getContent(), "\n")) {
```
Comment (kubawerlos) [inline src/Fixer/PhpTag/BlankLineAfterOpeningTagFixer.php]: Is it OK to check only fofr `\n` instead of RegEx with `\R`?

Comment (SpacePossum) [reply src/Fixer/PhpTag/BlankLineAfterOpeningTagFixer.php]: we support "\n" and "\r\n" but not "\r" only (old Mac thing I think?), so I think this is fine (we do this on other places as well)

**Comment by keradus** [comment]
(dropping milestone as this is WIP)

File: tests/Fixer/NamespaceNotation/BlankLineAfterNamespaceFixerTest.php [hunk 2]
```diff
@@ -138,6 +145,33 @@ class C {}\r",
...
+            ],
+            [
+                '<?php
+    namespace My\NS;
+
+    class X extends Y {}',
+            ],
+            [
+                '<?php
+namespace My\NS; // comment
```
Comment (julienfalque) [inline tests/Fixer/NamespaceNotation/BlankLineAfterNamespaceFixerTest.php]: What about adding a test case with a `/* comment */`?

Comment (SpacePossum) [reply tests/Fixer/NamespaceNotation/BlankLineAfterNamespaceFixerTest.php]: :+1:  added

**Comment by SpacePossum** [comment]
Thank you @kubawerlos.

**Pull request merged**